cannon/mipsevm/tests/evm_multithreaded_test.go

```diff
@@ -1193,6 +1193,32 @@ func TestEVM_UnsupportedSyscall(t *testing.T) {
 	}
 }
 
+func TestEVM_EmptyThreadStacks(t *testing.T) {
+	t.Parallel()
+	var tracer *tracing.Hooks
+
+	cases := []struct {
+		name            string
+		activeStackSize int
```

Comment:
We can cut this as a testCase param if it is always 0 (and just hardcode the 0 in the test itself):
```suggestion
```

Reply:
sure